deploy/runtools/runtime_config.py

```diff
@@ -65,13 +69,374 @@ def get_local_driver_path(self):
+            "/usr/lib64/gconv/ISO8859-9.so",
+            "/usr/lib64/gconv/ISO8859-9E.so",
+            "/usr/lib64/gconv/ISO_10367-BOX.so",
+            "/usr/lib64/gconv/ISO_11548-1.so",
+            "/usr/lib64/gconv/ISO_2033.so",
+            "/usr/lib64/gconv/ISO_5427-EXT.so",
+            "/usr/lib64/gconv/ISO_5427.so",
+            "/usr/lib64/gconv/ISO_5428.so",
+            "/usr/lib64/gconv/ISO_6937-2.so",
+            "/usr/lib64/gconv/ISO_6937.so",
+            "/usr/lib64/gconv/JOHAB.so",
+            "/usr/lib64/gconv/KOI-8.so",
+            "/usr/lib64/gconv/KOI8-R.so",
+            "/usr/lib64/gconv/KOI8-RU.so",
+            "/usr/lib64/gconv/KOI8-T.so",
+            "/usr/lib64/gconv/KOI8-U.so",
+            "/usr/lib64/gconv/LATIN-GREEK-1.so",
+            "/usr/lib64/gconv/LATIN-GREEK.so",
+            "/usr/lib64/gconv/MAC-CENTRALEUROPE.so",
+            "/usr/lib64/gconv/MAC-IS.so",
+            "/usr/lib64/gconv/MAC-SAMI.so",
+            "/usr/lib64/gconv/MAC-UK.so",
+            "/usr/lib64/gconv/MACINTOSH.so",
+            "/usr/lib64/gconv/MIK.so",
+            "/usr/lib64/gconv/NATS-DANO.so",
+            "/usr/lib64/gconv/NATS-SEFI.so",
+            "/usr/lib64/gconv/PT154.so",
+            "/usr/lib64/gconv/RK1048.so",
+            "/usr/lib64/gconv/SAMI-WS2.so",
+            "/usr/lib64/gconv/SHIFT_JISX0213.so",
+            "/usr/lib64/gconv/SJIS.so",
+            "/usr/lib64/gconv/T.61.so",
+            "/usr/lib64/gconv/TCVN5712-1.so",
+            "/usr/lib64/gconv/TIS-620.so",
+            "/usr/lib64/gconv/TSCII.so",
+            "/usr/lib64/gconv/UHC.so",
+            "/usr/lib64/gconv/UNICODE.so",
+            "/usr/lib64/gconv/UTF-16.so",
+            "/usr/lib64/gconv/UTF-32.so",
+            "/usr/lib64/gconv/UTF-7.so",
+            "/usr/lib64/gconv/VISCII.so",
+            "/usr/lib64/gconv/libCNS.so",
+            "/usr/lib64/gconv/libGB.so",
+            "/usr/lib64/gconv/libISOIR165.so",
+            "/usr/lib64/gconv/libJIS.so",
+            "/usr/lib64/gconv/libJISX0213.so",
+            "/usr/lib64/gconv/libKSC.so",
+            "/usr/lib64/libmemusage.so",
+            "/usr/lib64/libpcprofile.so",
+        ]
+
+        libs = list()
+        for dso in lddwrap.list_dependencies(Path(exe)):
+            if dso.soname is None:
+                assert '/ld-linux' in fspath(dso.path), f"dynamic linker is only allowed no soname, not: {dso}"
+                continue
+            if 'linux-vdso.so' in dso.soname:
+                continue
+            assert dso.path is not None, f"{dso.soname} has no linkage reported by ldd for {exe} in: {dso}"
+            if dso.path in glibc_shared_libs:
+                continue
+
+            # we do some moderate evilness here and collapse
+            # the version symlinks of our dso into the actual dso
+            # so that we have fewer pieces to copy over
+            libs.append((realpath(dso.path), dso.soname))
```

Comment:
@davidbiancolin this is the code that does the introspecting via `ldd` to figure out what to copy.  The contents of glibc could be derived dynamically by running what I documented on line 91 but I was reluctant to put rpm-specific stuff in the manager.  

Like we discussed, I'll post an alternate PR that accomplishes the same thing in a more portable way using `conda-build` and `constructor`.

Reply:
As an alternative to excluding the contents of glibc, we could look at the RPATH of the executable and only copy things that are coming from the RPATH (and whatever LD_LIBRARY_PATH is set to until we no longer need it)

Reply:
This code moved into `deploy/runtools/utils.py`.  I think it is fine to merge as is for now and possibly revisit in a future PR to simplify it.